### ISSUE TITLE
CoreCLR Linux Test builds are targeting Windows

### ISSUE THREAD

**Issue by MattGal** [dotnet/coreclr#15440]
Check Linux work item logs as seen here:
https://mc.dot.net/#/product/netcore/master/source/official~2Fcoreclr~2Fmaster~2F/type/test~2Ffunctional~2Fcli~2F/build/20171208.03/workItem/CoreMangLib.components.XUnitWrapper/wilogs

Basically the tests are failing due to apparently targeting Windows during test build. 

```
2017-12-08 14:48:26,992: INFO: io(27): copy_tree_to: Copying '/home/helixbot/dotnetbuild/work/ac303c3b-d0ae-47d2-9bea-55271f83ec8d/Work/88609d5a-fc78-455e-b8bb-8f430e0a0843/Unzip/runtests.cmd' to '/home/helixbot/dotnetbuild/work/ac303c3b-d0ae-47d2-9bea-55271f83ec8d/Work/88609d5a-fc78-455e-b8bb-8f430e0a0843/Exec/runtests.cmd'
...
2017-12-08 14:48:26,998: INFO: executor(103): run: Successfully downloaded work item payload
2017-12-08 14:48:26,999: INFO: executor(381): _execute_command: running chmod +x $HELIX_WORKITEM_PAYLOAD/runtests.sh && $HELIX_PYTHONPATH $HELIX_CORRELATION_PAYLOAD/RunnerScripts/scriptrunner/scriptrunner.py --script $HELIX_WORKITEM_PAYLOAD/runtests.sh in /home/helixbot/dotnetbuild/work/ac303c3b-d0ae-47d2-9bea-55271f83ec8d/Work/88609d5a-fc78-455e-b8bb-8f430e0a0843/Exec max 1800 seconds
```

**Comment by MattGal** [comment]
@RussKeldorph FYI.  Let me know if whoever takes this needs assistance in their investigation.

**Comment by jashook** [comment]
Fixed with #15815
